ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -370,6 +552,33 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-fullyprivate-proxy
+- as: azure-ipi-ingress-controller-arm-mixarch-f28
```

Comment:
Similar comments with https://github.com/openshift/release/pull/54740#discussion_r1698220768 and it is duplicated with `azure-ipi-ingress-custom-controller-arm`

Reply:
It's not a disconnected cluster, and we don't have same job in amd64 and arm64 arch, so I'd prefer to remove `azure-ipi-ingress-custom-controller-arm-f28`, thought? @jianlinliu

Reply:
ok, i see there is `azure-ipi-ingress-custom-controller-fips-amd-f28`, so I will remove `azure-ipi-ingress-controller-arm-mixarch-f28` and `azure-ipi-ingress-custom-controller-arm-f28`.

Reply:
@JianLi-RH which one did you keep? In e2e, azure-ipi-ingress-custom-controller-fips-amd-f28 is a destructive job.

```
openshift-openshift-tests-private-release-4.16__multi-nightly.yaml:- as: azure-ipi-ingress-custom-controller-fips-amd-f9-destructive
```

@jianlinliu In e2e, we have

```
openshift-openshift-tests-private-release-4.16__multi-nightly.yaml:- as: azure-ipi-ingress-controller-arm-mixarch-f5
openshift-openshift-tests-private-release-4.16__multi-nightly.yaml:- as: azure-ipi-ingress-custom-controller-arm-f5
```
They are dupe and we should drop one of them, right?

Reply:
Personally I never exclude destructive ci jobs when picking up install profile for upgrade ci. There are 4 ci jobs using the same install workflow:
```
$ grep -r "as: azure-ipi-ingress-.*controller" *release-4.16*
openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-upgrade-from-stable-4.15.yaml:- as: azure-ipi-ingress-custom-controller-fips-amd-f28
openshift-openshift-tests-private-release-4.16__multi-nightly.yaml:- as: azure-ipi-ingress-controller-arm-mixarch-f5
openshift-openshift-tests-private-release-4.16__multi-nightly.yaml:- as: azure-ipi-ingress-controller-amd-mixarch-f9-destructive
openshift-openshift-tests-private-release-4.16__multi-nightly.yaml:- as: azure-ipi-ingress-custom-controller-arm-f5
openshift-openshift-tests-private-release-4.16__multi-nightly.yaml:- as: azure-ipi-ingress-custom-controller-fips-amd-f9-destructive
```
2 jobs are a pair for a full coverage for a specific configuration. From my understanding, the above 4 jobs should be two pair, and one pair seem like duplicated with the other pair. Once drop one pair from e2e ci, then upgrade ci need to pick up one install profile from the remaining pair of e2e ci jobs (including non-destructive and destructive jobs)

Reply:
Sorry, just see @shellyyang1989 's comment, let me replace `azure-ipi-ingress-custom-controller-fips-amd-f28` with `azure-ipi-ingress-custom-controller-arm` to keep e2e and upgrade have same jobs.